src/browzine-summon-adapter.js

```diff
@@ -57,21 +81,21 @@ angular.module("summonApp.directives")
   };
 
   function shouldEnhance(scope) {
-    return (isJournal(scope) && getIssn(scope)) || (isArticle(scope) && getDoi(scope));
+    return !!((isJournal(scope) && getIssn(scope)) || (isArticle(scope) && getDoi(scope)));
```

Comment:
Is the `!!` necessary here? I think the `&&`s and `||`s will case it into a boolean before it gets to this operator.

Reply:
My exact thoughts too until I unit tested it and an issn returned, Eeek...! 🎃
https://github.com/thirdiron/browzine-summon-adapter/pull/13/files#diff-18e41204f346c94301e808d824658669R71

Reply:
2spooky4me